lookup_tables/greynoise/advanced/riot_advanced.yml

```diff
@@ -102,6 +102,19 @@ LogTypeMap:
         - "LocalAddressIP6"
         - "RemoteAddressIP4"
         - "RemoteAddressIP6"
+    - LogType: Crowdstrike.FDREvent
+      Selectors:
+        - 'p_any_ip_addresses'
+        - '$.event.CurrentLocalIP'
```

Comment:
Same observation as above with regard to IP field selectors.

Reply:
Removed.